PovioKit/Classes/Networking/RestClient/Host.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Host.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 14/01/2019.
+//  Copyright © 2019 Povio Labs. All rights reserved.
+//
+
+import Foundation
+
+public enum Host {
+  case development
+  case production
+}
+
+public extension Host {
+  var baseURL: String {
+    switch self {
+    case .development:
+      return "https://us-central1-waybots-a2558.cloudfunctions.net"
+    case .production:
+      return "https://us-central1-waybots-production.cloudfunctions.net"
```

Comment:
We are using fixed hosts here. This should go away. Also, we should think about making this generic enough so implementation can define `baseUrl` on its own.